scipy/optimize/tests/test__differential_evolution.py

```diff
@@ -4,13 +4,14 @@
 from multiprocessing.dummy import Pool as ThreadPool
 import platform
 import warnings
+from functools import partial
```

Comment:
`partial` is not used in this file. Maybe you added this line to the wrong file?